nexus/db-queries/src/db/datastore/mod.rs

```diff
@@ -641,10 +651,12 @@ mod test {
             renewed.console_session.time_last_used > session.time_last_used
         );
 
+        // TODO: check the opctx on these changes, make sure we're using the
+        // right thing between opctx or authn_opctx
```

Comment:
does this need to be addressed before merging the PR, or is that something you intend to test later?